src/v/cluster/cluster_utils.h

```diff
@@ -40,6 +40,23 @@ patch<broker_ptr> calculate_changed_brokers(
 
 /// Creates the same topic_result for all requests
 // clang-format off
+template<typename T>
+CONCEPT(requires requires(const T& req) {
+    { req.name } -> std::convertible_to<const model::topic_namespace&>;
+})
+// clang-format on
+std::vector<topic_result> create_non_replicable_topic_results(
```

Comment:
This method is nearly identical to the one beneath it. You can reduce code duplication by making a common method that accepts a functor that performs the diff between the two.